Makefile

```diff
@@ -17,6 +17,8 @@ generate:
 	hack/update-codegen.sh
 .PHONY: generate
 
-update-deps:
-	hack/update-deps.sh
-.PHONY: update-deps
+vendor:
```

Comment:
Documenting is fine, but we don't have that anywhere coded. I'd rather we don't here, either.

Reply:
do you mean having a `make vendor`? We use it in our container tools repos, but if we don't want it here I can just document the three steps in the README instead, i.e
```
go mod tidy
go mod vendor
go mod verify
```
wdyt?

Reply:
Yeah, that sounds good.

Reply:
Fixed